### DISCUSSION TITLE
Workflow απορία-ες

### DISCUSSION THREAD

**Discussion by dimidimots** [sdy60/syllabus#16] [Q&A] [answered]
Καλησπέρα, 

Μια μικρή ερώτηση. Πόσα pull request μπορώ να τρέχω ταυτόχρονα? Γιατί νομίζω,ότι επιτρέπει μόνο ένα?

**Comment by epidrome** [comment, marked as answer]
θεωρητικά, όσα θες, πάντως,
δύσκολο να απαντηθεί η ερώτηση αυτή,
χωρίς να έχουμε κάποιο παράδειγμα
